### DISCUSSION TITLE
Stability of numerical solutions

### DISCUSSION THREAD

**Discussion by GenB31415** [zwicker-group/py-pde#105] [Q&A]
How one can in py-pde control the stability of numerical solutions also with the standard approaches, e.g. Von Neumann stability analysis
(https://en.wikipedia.org/wiki/Von_Neumann_stability_analysis) 
or Courant–Friedrichs–Lewy condition
https://en.wikipedia.org/wiki/Courant%E2%80%93Friedrichs%E2%80%93Lewy_condition

Thank you

**Comment by david-zwicker** [comment]
I'm not sure I understand what you expect the package to do. You could of course open a feature request under "Issues" to give details about the suggest functions. Even better, you could also try to implement it yourself!

**Comment by GenB31415** [comment]
Okay, I have modeled the dynamic behavior of an advanced system of nonlinear complex equations as an extension of the simple models discussed in the py-pde package. It turned out that, in particular, the found numerical solutions reproduce the conservation laws in the system with very high accuracy and also its agree well with the known criteria for the numerical stability. Many thanks to the author for creation this must-have package and also for the high-quality documentation.
